css/style.css

```diff
@@ -1,19 +1,157 @@
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+body, footer {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+/*Navigation*/
+.navbar{
+  background-color: white;
+  position: sticky;
```

Comment:
Does this rule work for you? It isn't working for me. Have you tried using `position: fixed` instead? 
```
    position: fixed;
    width: 100%;
```